pyproject.toml

```diff
@@ -37,8 +37,9 @@ numpy = "^1.20.3"
 opencv-python-headless = "^4.5.1"
 scikit-image = "^0.18.1"
 scikit-learn = "^0.24.2"
+# master branch after merge if `ClassifyImage` interface.
+smqtk-classifier = {git = "https://github.com/Kitware/SMQTK-Classifier.git", rev = "aa0dedf7929cbc3a0d71820691ddba450aea4198"}
```

Comment:
How stable do we expect `smqtk-classifier` to be? Do we anticipate any breaking changes that might have a downstream effect on `xaitk-saliency`?

Reply:
Oops! Good catch. This was added to track the branch that was developing the `ImageClassifier` interface before the release so that I could work on this branch. I should make a release of the smqtk-classifier package with the addition of that new interface and then fixup this dependency to reference a published version instead of this hash.

Reply:
In terms of "stability" the idea anyway is that this package, currently at least, specifies a min version as well as a pinned version in the `poetry.lock` file for consistent testing / dev-environment. If a new upstream version changes such that xaitk-saliency breaks, then we can more strictly lock-down the acceptable dependency version range. Or we could update xaitk-saliency to use newer versions when it is appropriate for our package to do so.

Reply:
Sounds good, just wanted to check this since the smqtk-classifier version was pinned to a specific git hash instead of a version number like the other packages.